src/window.js

```diff
@@ -17,6 +17,7 @@ define([
             preventDoubleScrolling(this.$f.find('.fd-scrollable'));
             $(window).resize(adjustToWindow);
             $(document).scroll(adjustToWindow);
+            $(document).ready(adjustToWindow);
```

Comment:
The reason tests failed is because this line needs to be moved down below `this.data.windowManager.offset = ...` (I also moved the two lines above it since they are similar). I moved them just above the line containing `this.adjustToWindow();` and I think if you do the same we might not have a merge conflict.